Formula/deno.rb

```diff
@@ -1,9 +1,9 @@
 class Deno < Formula
   desc "Command-line JavaScript / TypeScript engine"
   homepage "https://deno.land/"
-  url "https://github.com/denoland/deno/releases/download/v0.34.0/deno_src.tar.gz"
-  version "0.34.0"
-  sha256 "e6439e04b6df8db8d5192f98ee89c7d3ba9e966816fc7bf0d46cb52dc2e797aa"
+  url "https://github.com/denoland/deno/releases/download/v0.35.0/deno_src.tar.gz"
+  version "0.35.0"
```

Comment:
Homebrew is able to extract the version from the URL so this line isn't needed.

```
==> brew audit deno --online
==> FAILED
Error: 1 problem in 1 formula detected
deno:
  * Stable: version 0.35.0 is redundant with version scanned from URL
```

Reply:
🎉 Great that the version is no longer detected as `src` after the recent changes in `brew`. ✨ 

Also you might want to coordinate this with #50881.